honeycomb-kernels/src/grisubal/kernel.rs

```diff
@@ -125,6 +132,18 @@ pub fn build_mesh<T: CoordsFloat>(geometry: &Geometry2<T>, grid_cell_sizes: (T,
 
 // --- main kernels steps
 
+fn remove_redundant_poi<T: CoordsFloat>(geometry: &mut Geometry2<T>, (cx, cy): (T, T)) {
```

Comment:
I do not see this function as a `main kernel step`. Perhaps it should be moved in another part of the file.